init_test_db.sh

```diff
@@ -0,0 +1,6 @@
+psql -U postgres -c "create user aatams_test unencrypted password 'aatams_test';"
```

Comment:
I think there needs to be:

```
psql -U postgres -c "drop database aatams_test"
psql -U postgres -c "drop user aatams_test"
```

at the start here, if people want to run this script from the command line (and there might already be an `aatams_test` DB there).